docs/source/06_nodes_and_pipelines/03_modular_pipelines.md

```diff
@@ -4,406 +4,267 @@
-│   │   │   │   ├── pipeline.py
-│   │   │   │   ├── requirements.txt
-│   │   │   │   └── README.md
-│   │   │   └── __init__.py
-│   │   ├── __init__.py
-|   |   ├── cli.py
-│   │   ├── hooks.py
-│   │   ├── pipeline_registry.py
-│   │   ├── __main__.py
-|   |   └── settings.py
-│   ├── tests
-│   │   ├── __init__.py
-│   │   ├── pipelines
-│   │   │   ├── data_engineering
-│   │   │   │   ├── __init__.py
-│   │   │   │   └── test_pipeline.py
-│   │   │   ├── feature_engineering
-│   │   │   │   ├── __init__.py
-│   │   │   │   └── test_pipeline.py
-│   │   │   ├── modelling
-│   │   │   │   ├── __init__.py
-│   │   │   │   └── test_pipeline.py
-│   │   └── test_run.py
-│   ├── requirements.txt
-│   └── setup.py
-├── pyproject.toml
-├── README.md
-└── setup.cfg
-```
-</details>
-
-### Configuration
+<summary>Click here to see a worked example</summary>
 
-Nested configuration in modular pipelines is _not_ supported by Kedro. It means that putting config files (like `catalog.yml`) in `src/<python_package>/pipelines/<pipeline_name>/conf` will have no effect on the Kedro project configuration, however you may document it as a custom step that other users must complete as part of setting up your modular pipeline.
+```python
+cook_pipeline = Pipeline(
+    [
+        node(
+            func=defrost, 
+            inputs="frozen_veg", 
+            outputs="veg"
+        ), 
+        node(
+            func=grill, 
+            inputs="veg", 
+            outputs="grilled_veg"
+        )
+    ]
+)
 
-If you plan to manually hand off your modular pipeline to another project, you should document the configuration used by the pipeline in the `README.md` of your modular pipeline. For example, you may copy your configuration into the modular pipeline location before the pipeline hand off and instruct the users to copy `catalog.yml` into their top-level configuration:
+lunch_pipeline = Pipeline(
+    [
+        node(
+            func=eat, 
+            inputs="food", 
+            outputs=None
+        )
+    ]
+)
 
-```bash
-mkdir conf/base/catalog/  # create a separate folder for the pipeline configs
-cp src/<python_package>/pipelines/data_engineering/conf/catalog.yml conf/base/catalog/data_engineering.yml  # copy the pipeline configs
+cook_pipeline + lunch_pipeline 
 ```
 
-### Datasets
+This combination will visualise since it's valid pre-runtime, but it will not run since `food` is not an output of the cook pipeline:
 
-It is important to keep in mind that Kedro resolves the execution order of your pipeline's node based on their input and output datasets.
+![disjoined](../meta/images/cook_disjointed.png)
 
-For example, if `node1` outputs the dataset `A`, and `node2` requires the dataset `A` as an input, then `node1` is guaranteed to be executed before `node2` when Kedro runs the pipeline.
+* Combining `cook_pipeline + lunch_pipeline` will not work since because the `food` doesn't exist as an output of the `cook_pipeline`.
```

Comment:
```suggestion
* Combining `cook_pipeline + lunch_pipeline` will not work since `food` doesn't exist as an output of the `cook_pipeline`.
```

Can we actually remove this line, since it's repeated on L168?